fusor-ember-cli/app/templates/review/summary.hbs

```diff
@@ -14,9 +14,18 @@
 
     {{#if isOpenStack}}
         {{#accordion-item name="Red Hat Enterprise Linux - OpenStack Platform" isOpen=isOpenStackOpen}}
-            {{review-link label='Director URL' value=undercloudUrl isRequired=true isExternalURL=true}}
-            {{review-link label='Director username' value=undercloudUsername isRequired=true}}
-            {{review-link label='Director password' value=undercloudPassword isRequired=true}}
+            {{review-link label='Director URL'
+                          value=undercloudUrl
+                          isExternalURL=true}}
+            {{review-link label='Director username'
+                          value=undercloudUsername}}
+            {{review-link label='Director password'
+                          value=undercloudPassword}}
+            {{review-link label='Overcloud URL'
```

Comment:
@jmontleon, is this the label 'Overcloud URL' or is there another name?

I also added the password. Do we want it on the summary page?